src/Microsoft.Dnx.Tooling/Restore/ImplicitPackagesWalkProvider.cs

```diff
@@ -70,7 +75,11 @@ internal class ImplicitPackagesWalkProvider : IWalkProvider
             new RuntimeSpec("ubuntu-x64", "ubuntu", "linux-x64"),
 
             new RuntimeSpec("ubuntu.14.04", "ubuntu"),
-            new RuntimeSpec("ubuntu.14.04-x64", "ubuntu.14.04", "ubuntu-x64"));
+            new RuntimeSpec("ubuntu.14.04-x64", "ubuntu.14.04", "ubuntu-x64"),
+
```

Comment:
If you confirm for me that `dnu restore` works on CoreCLR on Ubuntu 15.04, I'm willing to add it. It would be similar to Linux Mint, just an unblocker, not formal support.